javascript/javascript1/week1/homework.js

```diff
@@ -0,0 +1,47 @@
+// future age
+let yearOfBirth;
+let yearFuture; 
+let date;
+prompt('Welcome to the software that can calculate your future age. If you want to continue please press OK','' );
+yearOfBirth = prompt('Enter you date of birth','' );
+yearFuture = prompt('Enter the year you want to know your age','' );
+date = yearFuture - yearOfBirth;
+document.write("In the year " + yearFuture + "  you will have " + date + " years");
+console.log("In the year " + yearFuture + "  you will have " + date + " years");
+
+//dog's age
+
+let dogYearOfBirth, dogYearFuture, shouldShowResultlnDogYear, dogYear;
+prompt('calculate the age of you Dog in the future. If you want to continue please press OK','' );
+dogYearOfBirth = prompt('Enter your dogs date of birth','' );
+dogYearFuture = prompt('Enter the year you want to know your dogs age','' );
+dogYear = dogYearFuture  - dogYearOfBirth;
+shouldShowResultlnDogYear =  dogYear * 7 ;
+document.write("In the year " + dogYearFuture + "  you dog will have " + dogYear + " human years "
++ " and " + shouldShowResultlnDogYear +" dog years " );
+console.log("In the year " + dogYearFuture + "  you dog will have " + dogYear + "human years"
++ " and " + shouldShowResultlnDogYear +" dog years ");
+
+//price houses
+
+let wide, deep, high, gardenZise, housePrinceFormulaA, housePrinceFormulaA1, housePrinceFormulaB, housePrinceFormulaC;
+// constants of the formula
+const a = 2.5, b = 1000, c = 300 ;  
+
+//house size variables
+prompt('calculate the price of your house, press ok to continue','' );
+wide = prompt('Enter the width you want','' );
+deep = prompt('Enter the deep you want','' );
+high = prompt('Enter the high you want','' );
+gardenZise = prompt('Enter the garden Zise you want','' );
+
+// house price formula
+housePrinceFormulaA = wide * deep * high ; 
```

Comment:
We have a small typo here
```suggestion
housePriceFormulaA = wide * deep * high ; 
```

Reply:
🤦🏽‍♂️